scss/xy-grid/_cell.scss

```diff
@@ -127,7 +127,7 @@
     // Base flex properties
     @include xy-cell-base($size);
 
-    @if($gutter-type == 'margin') {
+    @if($gutter-output == true && $gutter-type == 'margin') {
```

Comment:
@DanielRuf as mentioned above it has to be `and` here instead of `&&`

Reply:
Already done =)